eventkit_cloud/ui/static/ui/app/components/CreateDataPack/DataProvider.tsx

```diff
@@ -298,8 +298,8 @@ export class DataProvider extends React.Component<Props, State> {
                 >
                     <MapView
                         id={this.props.provider.id + "-map"}
-                        url={this.context.config.BASEMAP_URL}
-                        copyright={this.context.config.BASEMAP_COPYRIGHT}
+                        url={this.props.provider.preview_url}
```

Comment:
Are all providers guaranteed to have a preview url going forward? What happens when this isn't set?